apps/invite/views.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import status, viewsets
+
+from rest_framework.response import Response
+
+from rest_framework.permissions import IsAuthenticated
+
+from .models import Invitation
+
+from .serializers import InvitationSerializer
+
+class InvitationViewSet(viewsets.ModelViewSet):
```

Comment:
This view will still have the other endpoints because you using ModelViewset 

To confirm you can check this url "/doc" for the api documentation